src/main/java/io/fabric8/elasticsearch/plugin/KibanaUserReindexAction.java

```diff
@@ -168,38 +167,7 @@ public void apply(String action, ActionResponse response, @SuppressWarnings("raw
     private GetResult buildNewResult(GetResponse response) {
         String index = getIndex(response);
         String replacedIndex = response.getIndex();
-
-        // Check for .kibana.* in the source
-        BytesReference replacedContent = null;
-        if (!response.isSourceEmpty()) {
-            String source = response.getSourceAsBytesRef().toUtf8();
-            String replaced = source.replaceAll(replacedIndex, index);
-            replacedContent = new BytesArray(replaced);
-        }
-
-        // Check for .kibana.* in the fields
-        Map<String, GetField> responseFields = response.getFields();
```

Comment:
@portante The Nullpointer was here so it looks like response returns a null map.  I don't know under what condition this can occur. Its possible a bug in ES code, possibly a bug in SG code.  This refactoring essentially a defensive guard.